### DISCUSSION TITLE
Card collection

### DISCUSSION THREAD

**Discussion by JoshDe777** [Rapptz/discord.py#7279] [Q&A]
Hi, I'm looking to make a game with my bot where every player collects cards, similar to pokémon basically. For that, my first thought was to create an object for every player that registers with a .gameRegister or something... how can I do that? Or would it be more practical to add a node to a list for every user or something?

**Comment by itzsimpleboii8282** [comment]
You need to explain what you mean by "create an object for every player". Did you mean member/user objects? If yes, then author can be accessed with *message.author*, and while using ext.commands *ctx.author*...

**Reply by JoshDe777** [reply]
nah I want to create an object of the class "cardCollector" for every user that registers for the game..., so ideally it could be accessed with message.author.cardCollector ^^

**Reply by Miolus** [reply]
You could create an extra Player class
